src/components/AvatarWithIndicator.js

```diff
@@ -61,11 +59,14 @@ const AvatarWithIndicator = (props) => {
         isLarge ? styles.statusIndicatorLarge : styles.statusIndicator,
     ];
 
-    const hasPolicyMemberError = _.some(props.policiesMemberList, policyMembers => Policy.hasPolicyMemberError(policyMembers));
-    const hasCustomUnitsError = _.some(props.policies, policy => Policy.hasCustomUnitsError(policy));
+    const hasPolicyMemberError = _.some(props.policiesMemberList, policyMembers => PolicyUtils.hasPolicyMemberError(policyMembers));
     const hasPaymentMethodError = PaymentMethods.hasPaymentMethodError(props.bankAccountList, props.cardList);
     const hasWalletError = !_.isEmpty(props.userWallet.errors);
-    const shouldShowIndicator = hasPolicyMemberError || hasPaymentMethodError || hasWalletError || hasCustomUnitsError;
+    const hasAnyPolicyError = _.some(props.policies, policy => PolicyUtils.hasPolicyError(policy));
+    const hasCustomUnitsError = _.some(props.policies, policy => PolicyUtils.hasCustomUnitsError(policy));
+
+    // Show the indicator if there is some error.
+    const shouldShowIndicator = _.some([hasPolicyMemberError, hasPaymentMethodError, hasWalletError, hasAnyPolicyError, hasCustomUnitsError]);
```

Comment:
Just curious why do we use `_.some` instead of a plain `||` like what was done before?

Reply:
I know it was kind of unnecessary, but I think this will work better as we start to add more and more flows that can create errors under the settings. I can change it back if you feel strongly against it.

Reply:
^ same, low key didn't know you could even use `_.some` like this and I feel like `||` translates better to plain English kinda like you're saying:

We `shouldShowIndicator` when we `hasPolicyMemberError` or `hasPaymentMethodError` or `hasWalletError` or `hasAnyPolicyError`...

Reply:
Okay, I'll revert.

Reply:
Thanks -- I am not sure what "this will work better" means. I agree with @jasperhuangg that the version with `||` reads better (also, for JS noobs like me, I had to go verify what the method does to be sure, which adds time to understanding the code)